#!/usr/bin/env python
PACKAGE = "xtark_line_follower"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_low", int_t, 0, "HSV color space h_low", 0, 0, 255)
gen.add("s_low", int_t, 0, "HSV color space s_low", 70, 0, 255)
gen.add("v_low", int_t, 0, "HSV color space v_low", 50, 0, 255)
gen.add("h_high", int_t, 0, "HSV color space h_high", 10, 0, 255)
gen.add("s_high", int_t, 0, "HSV color space s_high", 255, 0, 255)
gen.add("v_high", int_t, 0, "HSV color space v_high", 255, 0, 255)

exit(gen.generate(PACKAGE, "xtark_ilne_follower", "HSV_reconfig"))
